client/src/components/NavBar/Notifications/NotificationCenter/NotificationCenter.tsx

```diff
@@ -23,7 +21,8 @@ const NotificationCenter = ({ loggedInUser, notifications }: NotificationProps):
   const handleClose = () => setAnchorEl(null);
 
   const countNotifications = () => {
-    return notifications.filter((item) => item.read === false && item.user === loggedInUser.email).length;
+    return notifications.filter((item: INotificationItem) => item.read === false && item.user === loggedInUser?.email)
```

Comment:
Just a bit confused here, `item.user === loggedInUser?.email` is this implying over the network we are sending a user everyone else's notifications too?  This would be a privacy concern if that's the case.

Reply:
I am not aware of what is happening in the Notifications, I just changed the settings to get the loggedInUser from the useAuth() instead of getting it as props.

Reply:
But I totally agree with your comment, we have to check on that.